### ISSUE TITLE
Make dir when lang contains folders

### ISSUE THREAD

**Issue by krns** [martinlindhe/laravel-vue-i18n-generator#25]
Hi, thanks a lot for providing this package. :)

I have multiple directories in the `resources/lang` directory, because my app serves different language files for each environment like:
```
resources/lang
resources/lang/en
resources/lang/en/backend
resources/lang/en/frontend
```

In the `en` directory are shared language files, in the `backend` and `frontend` directories are special language files. But if I use the `--multi` flag I get the following error:

`file_put_contents(.../public/js/lang/backend/auth.js): failed to open stream: No such file or directory`

This is because the `file_put_contents` method won't create the directories. 

What do you think about adding this feature?

**Comment by martinlindhe** [comment]
Absolutely welcome!
I think we overlooked mkdir of new paths when we added --multi support.
Since you have a working setup exposing this issue, would you be able to help us with a PR?

**Comment by krns** [comment]
Yey, I will add a PR this or next week.

**Comment by martinlindhe** [comment]
Released fix as 0.1.18 which should hit packagist any second now :-)
